### DISCUSSION TITLE
need support DataSpell

### DISCUSSION THREAD

**Discussion by Fei-Wang** [Homebrew/discussions#2643] [Casks] [answered]
JetBrains ide [DataSpell](https://www.jetbrains.com/dataspell/).

**Comment by SMillerDev** [comment, marked as answer]
https://www.jetbrains.com/help/dataspell/quick-start-guide.html or https://intellij-support.jetbrains.com/hc/en-us/?dataspell is where you get support for this JetBrains product

**Reply by Fei-Wang** [reply]
No, I don’t need DataSpell help, I want DataSpell to be added to cask just like PyCharm, and then brew install it

**Reply by SMillerDev** [reply]
See https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md for that
